db/schema.rb

```diff
@@ -11,10 +11,33 @@
 # It's strongly recommended that you check this file into your version control system.
 
 ActiveRecord::Schema.define(version: 2019_05_07_144554) do
```

Comment:
mmmm ... should the timestamp be different here?

Reply:
Is papertrail sneakily writing to the file instead of re-generating the schema?  Might be worth a manual regenerate if so to make sure it's different

Reply:
Schema regenerated